bot.py

```diff
@@ -85,11 +85,16 @@ async def start_new_game(update: Update, context: ContextTypes.DEFAULT_TYPE) ->
             )
             msg = await interactions.NEW_GAME(update, game)
             chat_id = update.message.chat.id
-            await context.bot.pin_chat_message(
-                chat_id,
-                msg.message_id,
-                True,
-            )
+            try:
+                await context.bot.pin_chat_message(
+                    chat_id,
+                    msg.message_id,
+                    True,
+                )
+            except Exception:
+                # log: can not pin
+                print("Cannot pin")
+                pass
```

Comment:
revert this code change since it was already changed in the previous branch on which you based yours.

you should have checkout from `v2` and not from `fix/bot-rights`

Reply:
resolved in last commit